test/NuGet.Tests.Apex/NuGet.Tests.Apex/Apex/NuGetUIProjectTestExtension.cs

```diff
@@ -26,7 +26,7 @@ public NuGetUIProjectTestExtension(ApexTestUIProject project, ITestLogger logger
             _logger = logger;
         }
 
-        public bool SeachPackgeFromUI(string searchText)
+        public bool SearchPackgeFromUI(string searchText)
```

Comment:
You fixed one typo, but not the other :D 

`SearchPack*a*ge`
I'd add a suggestion but you need to chnage the callers too so that wouldn't help.

Reply:
Haha how did I not see that!?